borg/lrucache.py

```diff
@@ -26,7 +30,10 @@ def __delitem__(self, key):
         self._lru.remove(key)
 
     def __contains__(self, key):
-        return key in self._cache
+        contains = key in self._cache
+        self._hits += contains
+        self._misses += not contains
```

Comment:
why is this here? usually one would expect that *getting* data is a hit and *not getting* data (from cache) is a miss. but here, simply checking is enough to increase the counters.